docs/userguide.rst

```diff
@@ -103,24 +103,24 @@ even some external service.
 
 An *event broker* delivers published events to all interested parties. It facilitates
 the cooperation between schedulers by notifying them of new or updated
-`schedules <schedule>`_ and `jobs <job>`_.
+:ref:`schedules <schedule>` and :ref:`jobs <job>`.
 
 .. _scheduler:
 
 A *scheduler* is the main interface of this library. It houses both a `data store`_ and
-an `event broker`_, plus one or more `job executors <job executor>`_. It contains
+an `event broker`_, plus one or more :ref:`job executors <job executor>`. It contains
 methods users can use to work with tasks, schedules and jobs. Behind the scenes, it also
 processes due schedules, spawning jobs and updating the next run times. It also
-processes available jobs, making the appropriate `job executors <job executor>`_ to run
+processes available jobs, making the appropriate :ref:`job executors <job executor>` to run
 them, and then sending back the results to the `data store`_.
 
 Running the scheduler
 =====================
 
 The scheduler_ comes in two flavors: synchronous and asynchronous. The synchronous
 scheduler actually runs an asynchronous scheduler behind the scenes in a dedicated
-thread, so if your app runs on :mod:`asyncio` or Trio_, you should prefer the
-asynchronous scheduler.
+thread, so if your app runs on :mod:`asyncio` or `Trio <https://trio.readthedocs.io/en/stable/>`_,
```

Comment:
Just add a counterpart to this later (`.. _Trio: https://trio.readthedocs.io/en/stable/`). I dislike inline links.

Reply:
fa0dba554264007896abd64fc42429e0fe658a47